uid2_client/input_util.py

```diff
@@ -119,3 +121,9 @@ def normalize_and_hash_phone(phone):
     if not is_phone_number_normalized(phone):
         raise ValueError("phone number is not normalized: " + phone)
     return get_base64_encoded_hash(phone)
+
+
+def get_datetime_utc_iso_format(timestamp):
```

Comment:
I checked how Python's @patch and call_args work together. Creating a single simple method might be easier for this unit tests.